### PR TITLE
Feature/rc 168 group feedbacks by article

### PR DESCRIPTION
**Pull request by GrimStal**

### CODE REVIEW

File: src/admin/apiAdminCommunication.ts [hunk 1]
```diff
@@ -101,3 +101,26 @@ function json(response) {
...
+			pagination += `limit=${limit}&`;
+		}
+		if (sort) {
+			pagination += `sort=${sort}&`;
+		}
+		if (sortOrder) {
+			pagination += `sortOrder=${sortOrder}&`;
+		}
+		if (pagination[pagination.length -1] === '&') {
+			pagination = pagination.slice(0, pagination.length -1);
```
Comment (x1n13y84issmd42) [inline src/admin/apiAdminCommunication.ts]: This is lame.

Comment (x1n13y84issmd42) [reply src/admin/apiAdminCommunication.ts]: A less lame approach would be having an array `pagination = [];`,  pushing pieces of the pagination query in it and returning `pagination.join('&')`;

Comment (GrimStal) [reply src/admin/apiAdminCommunication.ts]: You are right, I made almost the same in other code, but did not fix this one

File: src/admin/services/Api.ts [hunk 2]
```diff
@@ -85,5 +84,28 @@ const Api = {
...
+	 * Get article by ID
+	 */
+	getArticle: (id) =>
+		sendRequest(`/admin/api/articles/${id}`, 'GET'),
+
+	/**
+	 * Get feedbacks belong to article by ID
+	 */
+	getArticleFeedbacks: (id, page?, limit?, sort?, sortOrder?) => {
+		const pagination = setPagination(page, limit, sort, sortOrder);
```
Comment (tcat) [inline src/admin/services/Api.ts]: rename setPagination to something like getFormattedPagination or something without set.

**Pull request merged**